c2rust-transpile/src/lib.rs

```diff
@@ -19,6 +19,7 @@ use std::io::prelude::*;
 use std::path::{Path, PathBuf};
 use std::process;
 
+use compile_cmds::CompileCmd;
```

Comment:
Nit:

```suggestion
use crate::compile_cmds::CompileCmd;
```

Using `crate::` is helpful since it's portable to other non `lib.rs` files and it's clear it's not an external dependency.